hawc/apps/study/serializers.py

```diff
@@ -20,6 +22,25 @@ class Meta:
         fields = "__all__"
 
 
+class CreateStudySerializer(StudySerializer):
```

Comment:
this is the same as `SimpleStudySerializer` with a validate method; why not use that instead?